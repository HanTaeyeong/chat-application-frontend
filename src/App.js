import logo from './logo.svg';
import './App.css';

import { Route, Routes } from 'react-router-dom'

import { BrowserRouter } from 'react-router-dom';
import { createMemoryHistory } from 'history';

import Join from './components/Join/Join'
import Chat from './components/Chat/Chat'

const history = createMemoryHistory();

function App() {
  return (
    <BrowserRouter navigator={history} location={history.location} >
      <Routes>
        <Route path="/" exact element={<Join />} />
        <Route path="/chat" element={<Chat />} />
      </Routes>
    </BrowserRouter>
  );
}

export default App;
