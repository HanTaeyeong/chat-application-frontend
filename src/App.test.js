import { create, act } from 'react-test-renderer'
import App from './App';

test('App render test', () => {
  let app;
  act(() => {
    app = create(<App />)
  })
  expect(app.toJSON()).toMatchSnapshot()
})
