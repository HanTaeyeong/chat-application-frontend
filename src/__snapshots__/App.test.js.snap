// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render test 1`] = `
<div
  id="join-outer-container"
>
  <div
    className="join-inner-container"
  >
    <h1
      id="join-heading"
    >
      Join
    </h1>
    <input
      className="join-input"
      name="name"
      onChange={[Function]}
      onKeyPress={[Function]}
      placeholder="Name"
    />
    <input
      className="join-input"
      name="room"
      onChange={[Function]}
      onKeyPress={[Function]}
      placeholder="Room"
    />
    <div
      onClick={[Function]}
    >
      <button
        className="join-sign-in-button"
      >
        Sign In
      </button>
    </div>
  </div>
</div>
`;
